script.js

```diff
@@ -42,4 +110,55 @@ const closing = () => {
   menu.style.display = 'none';
 };
 
-closingbutton.addEventListener('click', closing);
\ No newline at end of file
+closingbutton.addEventListener('click', closing);
+
+document.getElementById('cardsid').innerHTML = data.map((array, index) => `
+<div class="card" id="cardsid_${index}">
+<h2 class="projecttitle">${array.name}</h2>
+<p class="projecttext">${array.description}</p>
+<ul class="buttonsproject">
+    <li class="button"><a class="smoke" href="#">${array.technology[0]}</a></li>
+    <li class="button"><a class="smoke" href="#">${array.technology[1]}</a></li>
+    <li class="button"><a class="smoke" href="#">${array.technology[2]}</a></li>
+</ul>
+<div class="cardbottom">
+    <input id="button_${index}" class="orangebottom hovereffect" type="button" value="See Project" onclick="portfoliopopup(${index})">
+</div>
+</div>
+`).join('');
+
+const portfoliopopup = (value) => {
+  popcontainer.innerHTML = `
+  <div id="blurry"></div>
+  <div class="popupcontainer">
+      <div class="topcontainer">
+          <h2 id="popid">${data[value].name}</h2>
+          <i id="popclosingbutton" class="fa-solid fa-xmark"></i>
+          <ul class="buttonsproject" id="popbuttonslist">
+              <li class="popbuttons"><a href="#">${data[value].technology[0]}</a></li>
+              <li class="popbuttons"><a href="#">${data[value].technology[1]}</a></li>
+              <li class="popbuttons"><a href="#">${data[value].technology[2]}</a></li>
+          </ul>
+      </div>
+      <div id="popcontainerid">
+          <img src="./img/pop.img.png" id="popimage" alt="popup image ten">
+          <div id="poptextbuttons">
+              <p id="poptext">
+                  ${data[value].description}
+              </p>
+              <div id="popbuttonsbottom">
+                  <button class="poporange">See Live <i class="fa-solid fa-share-from-square"></i></button>
+                  <button class="poporange">See Source <i class="fa-brands fa-github"></i></button>
+              </div>
+          </div>
+
+      </div>
+      
+  </div>`;
```

Comment:
- [ ] [REQUIRED] Good job for implementing changes requested by previous reviewers you are almost there :+1: But as mentioned early by the previous reviewers you need to stick as much as possible to the Figma template. Please use the same  `height` , remove extra  `spacing` and use the same position for your button on both mobile and desktop pop-up windows  as defined on the Figma template because is very important for the user to have a great user experience and for you to peer attention to details: smile

## Mobile 

#### Your mobile popup
![Screenshot from 2022-07-21 20-50-51](https://user-images.githubusercontent.com/6615394/180314086-1bf2e8f9-2d8b-41f9-bf40-1d9fc6cf6ad0.png)

#### Figma mobile popup
![Screenshot from 2022-07-21 20-49-50](https://user-images.githubusercontent.com/6615394/180314093-8703a208-2c58-4959-af36-d9f6581da068.png)

## Desktop
#### Your desktop popup
![Screenshot from 2022-07-21 20-49-33](https://user-images.githubusercontent.com/6615394/180314094-bb78351e-d97d-4319-9e18-ed7114f95da6.png)

#### Figma mobile popup
![Screenshot from 2022-07-21 20-49-13](https://user-images.githubusercontent.com/6615394/180314102-06979339-22aa-45c5-9478-3648a46af0c2.png)